### ISSUE TITLE
Using SequentialBuilder without an arch file

### ISSUE THREAD

**Issue by lunixbochs** [facebookresearch/flashlight#498]
### Feature Description
It would be nice to pass a vector of strings directly to a buildSequentialModule() variant, as this saves writing a temporary file to disk.

#### Additional Context
Here's a possible implementation: https://github.com/talonvoice/flashlight/commit/766f2fd847e7b83aedd69212a228007607ce768d

**Comment by tlikhomanenko** [comment]
We are planing to remove arch file at all and switch to plugins. What do you think about it?

**Comment by lunixbochs** [comment]
Hmm I like the arch layer lines, it would be much more annoying if I had to implement the whole sequentialbuilder logic myself.

**Comment by tlikhomanenko** [comment]
The problem now is that for example transformer forward should need to have extra param for forward like mask and if you do crazy stuff with resnet block with transformer inside this will not work anyway. Example of model definition with plugin looks like this https://github.com/facebookresearch/flashlight/blob/master/flashlight/app/lm/plugins/LmAdae512SinposL8H8Fc1024Dp03Ldp0Adsm.cpp (similar to pytorch/tensorflow style anyway).
